src/main/kotlin/no/nav/syfo/client/oppdfgen/PdlUtils.kt

```diff
@@ -0,0 +1,52 @@
+package no.nav.syfo.client.oppdfgen
+
+import no.nav.syfo.client.pdl.PdlClient
+import no.nav.syfo.client.pdl.domain.PdlHentPerson
+import no.nav.syfo.client.pdl.domain.isNotGradert
+import no.nav.syfo.client.pdl.domain.toPersonName
+import org.slf4j.LoggerFactory
+
+class PdlUtils(private val pdlClient: PdlClient) {
+
+    fun getPersonNameString(pdlPerson: PdlHentPerson?, fnr: String): String {
+        return pdlPerson?.toPersonName() ?: fnr
+    }
+
+    suspend fun getPersonAdressString(fnr: String): String? {
+        val personInfo = pdlClient.getPersonInfo(fnr)
+
+        if (personInfo != null && personInfo.isNotGradert()) {
+            val vegadresse = personInfo.hentPerson?.bostedsadresse?.first()?.vegadresse
+            val postnummer: String
+            val bosted: String
+            val adressenavn: String
+            val husnummer: String
+            val husbokstav: String
+
+            if (vegadresse != null) {
+                return if (!vegadresse.postnummer.isNullOrEmpty()) {
+                    adressenavn = vegadresse.adressenavn ?: ""
+                    husnummer = vegadresse.husnummer ?: ""
+                    husbokstav = vegadresse.husbokstav ?: ""
+                    postnummer = vegadresse.postnummer
+                    bosted = pdlClient.getPoststed(postnummer) ?: ""
+
+                    "$adressenavn ${husnummer}${husbokstav} $postnummer $bosted"
+                } else {
+                    log.info("Can not get person's postnummer due to postnummer is null or empty")
```

Comment:
Kan evnt vurdere å slå sammen noen av log-statementsa da alle uansett fører til null - usikker på om det vil gi oss nytteverdi å vite i detalje hva som gikk galt.

Mest for å unngå mange if/else